src/contrib/cluster/Akka.Cluster.Sharding.Tests/Akka.Cluster.Sharding.Tests.csproj

```diff
@@ -1,5 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\..\common.props" />
+    
+    <!-- use the xunit JSON settings file to disable parallel tests running-->
```

Comment:
Could not just reference `<Import Project="..\..\xunitSettings.props" />` here, because this `props` file references `..\..\xunit.runner.json` relative path, which is not valid here (need one more `..\`). Possibly there is some cleaner war do do that.

Reply:
Ah... I'm sure there is using some MsBuild variables, but good point. May as well just use collection fixtures then.